### DISCUSSION TITLE
Problem with pages routing in Nuxt3

### DISCUSSION THREAD

**Discussion by davidfrgla** [nuxt/nuxt#17518] [Questions] [answered]
Hi, I am facing a problem in local dev with routing.
Maybe I am doing something wrong.

I have created a folder in `pages` called `page` with a `[slug].vue` file so we have `/pages/page/[slug].vue`
The first few reloads work well but after a moment it redirects to 404 page.

Any clue ?

**Comment by FSL89** [comment]
I just started using Nuxt myself and realized that I need to restart the app everytime I make changes in the pages directory, otherwise the page is not present in the router.

**Reply by FSL89** [reply]
Maybe I should have read what you wrote more than twice before replying.

**Comment by davidfrgla** [comment, marked as answer]
The console says that:
`[Vue Router warn]: No match found for location with path "/requestProvider.js.map"`

**Reply by FSL89** [reply]
Maybe this thread can bring some ideas for troubleshooting:
https://github.com/vercel/next.js/discussions/35199

**Reply by FSL89** [reply]
What does it say in dev tools console? There should be a warning there with a stack of files reporting it.

**Reply by davidfrgla** [reply]
Nuxt console and devtool console does not show anything,
Actually, when in Chrome Private Navigation, I don't have the issue

**Reply by davidfrgla** [reply]
I have tried removing Coinbase Wallet but same problem

**Reply by FSL89** [reply]
I've tried searching for _requestProvider_ in all files of my current project repo, which currently is a pure Nuxt 3 RC app without any extra npm packages except for Tailwindcss and found nothing, which makes me think it's caused by a plugin or something.

Have you restarted the app after removing the Coinbase Wallet?

**Reply by davidfrgla** [reply]
I have tried in a new blank project with same routing, with Coinbase Wallet / MetaMask installed too and it is working fine
This is weird

**Reply by davidfrgla** [reply]
I was effectively due to `requestProvider.js.map` ;)
